Dockerfile

```diff
@@ -26,9 +30,11 @@ RUN apt-get update \
 WORKDIR /Annif
 RUN pip install --upgrade pip --no-cache-dir
 
-# Install all optional dependencies:
 COPY setup.py README.md LICENSE.txt projects.cfg.dist /Annif/
-RUN pip install .[dev,voikko,pycld3,fasttext,nn,omikuji,yake] --no-cache-dir
+# Install dependencies for optional features.
+ARG optional_dependencies=dev,voikko,pycld3,fasttext,nn,omikuji,yake
```

Comment:
ARG is now declared twice? Isn't once enough?

Reply:
Once is not enough; each stage needs have its own declaration: https://docs.docker.com/engine/reference/builder/#scope

Reply:
Ah, OK. Thanks for the clarification!